_includes/codepens/editor-dfree/style.css

```diff
@@ -1,10 +1,12 @@
+body {
```

Comment:
This will potentially change the fonts of the entire page, so maybe use `.mce-content-body` instead?